test/unit/Flicking.spec.ts

```diff
@@ -638,6 +638,36 @@ describe("Flicking", () => {
         expect(clickSpy.calledOnce).to.be.true;
       });
 
+      it("should trigger click event when Flicking's dragged under 1px", async () => {
+        const flicking = await createFlicking(El.DEFAULT_HORIZONTAL, { preventClickOnDrag: true, inputType: ["pointer"] });
+        const clickSpy = sinon.spy();
+        const testPanel = flicking.panels[0];
+
+        testPanel.element.addEventListener("click", clickSpy);
+
+        flicking.element.dispatchEvent(new PointerEvent("pointerdown", {
+          pointerId: 1,
+          buttons: 1,
+          clientX: 0,
+          clientY: 0,
+          cancelable: true,
+        }));
+
+        window.dispatchEvent(new PointerEvent("pointermove", {
+          pointerId: 1,
+          buttons: 1,
+          clientX: 0.5,
+          clientY: 0,
+          movementX: 0,
+          movementY: 0,
+          cancelable: true,
+        }));
```

Comment:
Since this is a reproduction of a special case of input, I think it's okay to include a check for the Flicking's event of new test. 
This events fires `holdStart`, but `move` event should not fire. Before this fix was applied, the `move` event was firing in this case.

Reply:
I just mentioned that because it's kinda tricky to handle input events in the test environment.
As some events just don't work at all, it's a good thing you've added a TC of that.